frontend/react/style-guide.md

```diff
@@ -799,9 +824,9 @@ src
 
     > Why? Lifecycle methods should be class functions, not instance functions.
 
-    ```jsx
+    ```tsx
     // bad
-    class MyComponent extends Component {
+    function MyComponent() {
       componentDidMount = () => {
```

Comment:
I don't know if we should remove this section (because we are not using class components anymore, only in legacy projects, and even those should be migrated sooner or later) or rework it to use `useEffect` since lifecycle methods do not apply to functional components

Reply:
I believe that it would be better to leave it as a class component because it could be useful for someone working on a legacy project with class components

Reply:
I believe that it would be better to leave it as a class component because it could be useful for someone working on a legacy project with class components